src/utils/manager.py

```diff
@@ -105,22 +197,24 @@ def install() -> None:
     Raises:
         Error: Raised if operation to install NFS common package fails.
     """
-    _logger.debug("Installing `nfs-common` package from apt archive")
-    try:
-        apt.update()
-        apt.add_package("nfs-common")
-    except (apt.PackageError, apt.PackageNotFoundError) as e:
-        _logger.error(f"Failed to install `nfs-common` package. Reason\n:{e.message}")
-        raise Error(e.message)
+    apt.update()
+    for pkg in ["nfs-common", "autofs"]:
+        _logger.debug(f"Installing `{pkg}` package from apt archive")
+        try:
+            apt.add_package(pkg)
+        except (apt.PackageError, apt.PackageNotFoundError) as e:
+            _logger.error(f"Failed to install `{pkg}` package. Reason\n:{e.message}")
+            raise Error(e.message)
 
 
 def remove() -> None:
     """Remove NFS utilities for mounting NFS shares."""
-    _logger.debug("Removing `nfs-common` package from system packages")
-    try:
-        apt.remove_package("nfs-common")
-    except apt.PackageNotFoundError:
-        _logger.warning("`nfs-common` package not found on system. Doing nothing")
+    for pkg in ["nfs-common", "autofs"]:
```

Comment:
Again, not covered in unit test. I know it wasn't before either, but it'd be good to cover this while you touch it.